v3/src/components/free-tile-row.tsx

```diff
@@ -8,13 +9,14 @@ import { CodapComponent } from "./codap-component"
 import "./free-tile-row.scss"
 
 interface IFreeTileRowProps {
+  content?: IDocumentContentModel
   row: IFreeTileRow
   getTile: (tileId: string) => ITileModel | undefined
 }
-export const FreeTileRowComponent = observer(({ row, getTile }: IFreeTileRowProps) => {
+export const FreeTileRowComponent = observer(({ content, row, getTile }: IFreeTileRowProps) => {
   const handleCloseTile = (tileId: string) => {
     if (!tileId) return
-    row?.removeTile(tileId)
+    content?.deleteTile(tileId)
```

Comment:
I think for now you probably have to do both:
```typescript
    content?.deleteTile(tileId)
    row?.removeTile(tileId)
```
With some additional work we could make it the case that the row models respond to content changes, but that's not in place right now, so manually keeping the two in sync will presumably be required for the time being.

Reply:
Doesn't this do the same thing? 
https://github.com/concord-consortium/codap/blob/849725659143f07758ca41f1b448ef5ad767c00b/v3/src/models/document/document-content.ts#L168

Reply:
Right you are! My bad. 🤦